osu.Game/Configuration/SettingSourceAttribute.cs

```diff
@@ -31,7 +34,15 @@ public class SettingSourceAttribute : Attribute, IComparable<SettingSourceAttrib
 
         public int? OrderPosition { get; }
 
-        public SettingSourceAttribute(string label, string description = null)
+        /// <summary>
+        /// The type of the settings control which handles this setting source.
+        /// </summary>
+        /// <remarks>
+        /// Must be a type deriving <see cref="SettingsItem{T}"/> with a public constructor.
```

Comment:
to complete it:
```suggestion
        /// Must be a type deriving <see cref="SettingsItem{T}"/> with a public parameterless constructor.
```

Reply:
The constructor part is actually unnecessary.

Reply:
oh? you'll need a public parameterless constructor to instantiate this type with `Activator.CreateInstance` though, unless you mean it's pretty obvious that it doesn't need to be mentioned?

Reply:
Yeah you're right, I was thinking it's pretty obvious but I'll add it back in for safety...